include/mxnet/tensor_blob.h

```diff
@@ -36,8 +36,15 @@
 #include <utility>
 #include <algorithm>
 #include "./base.h"
+
 namespace mxnet {
 
+// redefine DLPack enumeration to be backward compatible.
+const int kCPU = kDLCPU;
+const int kGPU = kDLGPU;
+// extension type code under TVM function.
+const int kTVMNDArrayTypeCode = 19;
```

Comment:
would it make sense to make it an enumerator?

Reply:
mostly because the 19 seems arbitrary? and maybe extensible to other numbers in the future? in that case, an enum could help to manage accidental overlap.
although my assumptions here may not be correct.

Reply:
This enumerator is allocated in the TVM side and reserved for MXNet and NNVM project, so it is not arbitrary chosen https://github.com/dmlc/tvm/blob/master/include/tvm/runtime/c_runtime_api.h#L97

Reply:
It is picked to be the last reserved enumerator for NNVM

Reply:
will add a comment about this in the updated code